modules/subdyn/src/SD_FEM.f90

```diff
@@ -853,7 +926,39 @@ SUBROUTINE SetElementProperties(Init, p, ErrStat, ErrMsg)
          p%ElemProps(i)%Iyy    = Iyy
          p%ElemProps(i)%Jzz    = Jzz
          p%ElemProps(i)%Shear  = Shear
-         p%ElemProps(i)%kappa  = kappa
+         p%ElemProps(i)%kappa_x  = kappa
+         p%ElemProps(i)%kappa_y  = kappa
```

Comment:
The code appears to be inconsistent whether we use `kappa` or `Kappa` (not your fault). This pull request is our chance to change that everywhere since you are changing this variable name anyway. Let's use upper case `Kappa` (`Kappa_x` and `Kappa_y`)  to be consistent with the other fields already defined in the registry. 

Please only change the variable `kappa`, not the other ones (as it would create noise in the pull request).

Reply:
Sure